src/test/java/nextstep/ladder/domain/player/NamesTest.java

```diff
@@ -0,0 +1,33 @@
+package nextstep.ladder.domain.player;
+
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.DisplayName;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.params.ParameterizedTest;
+import org.junit.jupiter.params.provider.ValueSource;
+
+import static org.assertj.core.api.Assertions.assertThat;
+
+@DisplayName("Names 테스트")
+class NamesTest {
+
+    Names names;
```

Comment:
```suggestion
    private Names names;
```

테스트 코드라서 큰 의미는 없더라도 항상 접근제어자를 최대한 좁게 설정하는 것은 습관화해 주세요. 😉